prow/cmd/gerrit/main.go

```diff
@@ -147,3 +191,23 @@ func main() {
 		return cfg().Gerrit.TickInterval.Duration
 	})
 }
+
+// helper to update disk metrics (copied from ghproxy)
+func diskMonitor(interval time.Duration, diskRoot string) {
```

Comment:
Nit: Ideally we wouldn't have to copy/paste each time we want to do this for every component.